content/docs/secure/encryption/_index.md

```diff
@@ -117,6 +117,10 @@ Access to the data will be lost for ever.
 
 Refer to [Vault Configuration section](vault) for minimal configuration steps required to support Encryption and other configuration considerations.
 
+## Key Rotation (rekey)
+This preview of encryption includes the ability to change the KEK of an encrypted volume, an operation commonly known as Shallow Rekey, or 

```

Comment:
I would put "Key Encryption Key (KEK)" for the first occurrence and then use KEK in the rest of the documentation.

Reply:
sounds good.

Reply:
@AronAtDell I have addressed this change in the latest edits. Please review rekey.md as this file is new and contains the brunt of the changes for rekey documentation. Thank you.